plat/arm/css/sgi/sgi_bl31_setup.c

```diff
@@ -9,6 +9,21 @@
 #include <plat_arm.h>
 #include <sgi_plat_config.h>
 #include <sgi_ras.h>
+#include "../../css/drivers/scmi/scmi.h"
```

Comment:
Can we just add the path to the makefile to make the path not relative, as we might well reorganize this code at a later date (maybe PLAT_INCLUDES or similar)?